### ISSUE TITLE
 Hyperparameter Tuning

### ISSUE THREAD

**Issue by christine-a11y** [christine-a11y/cassava-leaf-disease-classification#25]
<html>
<body>
<h3 data-start="7309" data-end="7322" class="PDq2pG_selectionAnchorContainer"> Նպատակ<span aria-hidden="true" class="PDq2pG_selectionAnchor"></span></h3>
<p data-start="7323" data-end="7364">Գտնել լավագույն training configuration-ը։</p>
<h3 data-start="7366" data-end="7379">Փորձարկել</h3>
<ul class="contains-task-list" data-start="7381" data-end="7479">
<li class="task-list-item" data-start="7381" data-end="7400">
<input disabled="" type="checkbox"> Learning rate
</li>
<li class="task-list-item" data-start="7401" data-end="7417">
<input disabled="" type="checkbox"> Batch size
</li>
<li class="task-list-item" data-start="7418" data-end="7433">
<input disabled="" type="checkbox"> Optimizer
</li>
<li class="task-list-item" data-start="7434" data-end="7452">
<input disabled="" type="checkbox"> Weight decay
</li>
<li class="task-list-item" data-start="7453" data-end="7466">
<input disabled="" type="checkbox"> Dropout
</li>
<li class="task-list-item" data-start="7467" data-end="7479">
<input disabled="" type="checkbox"> Epochs
</li>
</ul>
<h3 data-start="7481" data-end="7493">⚠️ Կանոն</h3>
<p data-start="7495" data-end="7535">Յուրաքանչյուր experiment պետք է գրանցվի։</p>
<div class="group TyagGW_tableContainer"><div tabindex="-1" class="TyagGW_tableWrapper flex flex-col-reverse w-fit">
Experiment | Change | Result
-- | -- | --
Exp 1 | Baseline | ...
Exp 2 | LR change | ...
Exp 3 | Batch size | ...

</div></div>
<h3 data-start="7670" data-end="7685" class="">Deliverable</h3>
<p data-start="7687" data-end="7713">Experiment tracking table</p>
</body>
</html>
